command/daemon/command.go

```diff
@@ -8,11 +8,17 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/giantswarm/microkit/command/daemon/flag"
 	microerror "github.com/giantswarm/microkit/error"
+	microflag "github.com/giantswarm/microkit/flag"
 	"github.com/giantswarm/microkit/logger"
 	"github.com/giantswarm/microkit/server"
 )
 
+var (
+	Flag = flag.New()
```

Comment:
Each `flag` package provides a `New` method to get a new initialized flag structure.